firmware/quintuna/VC/src/app/app_efuse.c

```diff
@@ -0,0 +1,49 @@
+#include "app_efuse.h"
+
+#include <stdbool.h>
+#include <stddef.h>
+
+#include "io_efuse.h"
+#include "app_canTx.h"
+
+static void (*const efuse_enabled_can_setters[NUM_EFUSE_CHANNELS])(bool) = {
```

Comment:
we got std::array now 😭 we made it out the mud

Reply:
also I would group by efuse object rather than group by similar functionality

Reply:
this also has a nice side effect of improving data locality and making code actually ok to work with